docs/install/install_everest_helm_charts.md

```diff
@@ -89,23 +101,54 @@ Here are the steps to install Percona Everest and deploy additional database nam
                 everest   LoadBalancer   10.43.172.194   34.175.201.246       8080:8080/TCP    10s
                 ```
 
+        3. Run the following command to connect to Percona Everest when **TLS is enabled**:       
+                    
+            ```sh
+            kubectl patch svc/everest -n everest-system -p '{"spec": {"type": "LoadBalancer"}}'
+            ```
+
+            ??? example "Expected output"
+                ```sh 
+                kubectl get svc/everest -n everest-system
+                ```
+                ```
+                NAME      TYPE           CLUSTER-IP      EXTERNAL-IP     PORT(S)          AGE
+                everest   LoadBalancer   10.43.172.194   34.175.201.246       8080:443/TCP    10s
```

Comment:
Are you sure it is `8080:443`? It should be `443:8080`, but please install once with TLS and verify